deps.edn

```diff
@@ -1,60 +1,59 @@
 {:paths ["src" "resources"]
  :deps
-        {
```

Comment:
sorry for all the whitespace change but it was aligned before, so I had to realign everything after the few changes, which were simply adding extra namespaces to libraries that were missing it.